core/engine/src/value/conversions/mod.rs

```diff
@@ -39,6 +39,24 @@ impl From<char> for JsValue {
     }
 }
 
+impl From<&str> for JsValue {
+    #[inline]
+    fn from(value: &str) -> Self {
+        let _timer = Profiler::global().start_event("From<&str>", "value");
+
+        Self::from(js_string!(value))
+    }
+}
+
+impl From<String> for JsValue {
+    #[inline]
+    fn from(value: String) -> Self {
+        let _timer = Profiler::global().start_event("From<&str>", "value");
+
+        Self::from(JsString::from(value))
+    }
+}
+
```

Comment:
Uhh I'm slightly worried about this expression. If we add it, then all calls to `js_error` will fallback to runtime conversion to `UTF-16`, instead of using the nifty `js_str` which automatically converts to UTF-16 at compile time.

Reply:
That was one of the reasons of removing `From<&str> for JsValue`

Reply:
I'm not sure there's another way? Outside of.. not implementing it I guess.